src/main/java/edu/rosehulman/covidtracer/model/SystemUser.java

```diff
@@ -24,12 +24,12 @@
 
 	@ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
 	@JoinColumn(name = "role_id", nullable = false)
-	private Role role;
+	private UserRole role;
 
 	public SystemUser() {
 	}
 
-	public SystemUser(Person person, String password, String salt, Role role) {
+	public SystemUser(Person person, String password, String salt, UserRole role) {
```

Comment:
Didn't we say we are getting rid of this table? We don't need to store passwords because of rosefire.